src/main/java/org/fundacionjala/coding/lizzy/Sort.java

```diff
@@ -0,0 +1,28 @@
+package org.fundacionjala.coding.lizzy;
+
+public final class Sort {
+    private Sort() {
+
+    }
+
+    public static String sortTheInnerContent(final String sentence) {
+        String[] words = sentence.split(" ");
+        StringBuilder sorted = new StringBuilder();
```

Comment:
You can use a StringJoiner with delimiter blank space. 
so the concatenation on line 24 and the trim on line 26 will no longer necessary.
```java
    public static String sortTheInnerContent(final String sentence) {
        String[] words = sentence.split(" ");
        StringJoiner sorted = new StringJoiner(" ");
        for (String word: words) {
            String first = word.substring(0, 1);
            String last = word.substring(word.length() - 1);
            String[] middle = word.substring(1, word.length() - 1).split("");
            Arrays.sort(middle, Comparator.reverseOrder());
            sorted.add(first + String.join("", middle) + last);
        }
        return sorted.toString();
    }
```